src/lab/Graphs/LineAreaGraph/styles.ts

```diff
@@ -149,5 +156,57 @@ const usePlotLineAreaGraphStyles = makeStyles((theme: Theme) => ({
     strokeOpacity: 0.2,
   },
 }));
+const TooltipMui = withStyles((theme: Theme) => ({
```

Comment:
This can be moved to its own tsx file like TooltipMui.tsx
Since this is a component

Reply:
sure

Reply:
Done